### PR TITLE
Add Slack keywords and reminders

### PR DESCRIPTION
**Pull request by macraig**

### CODE REVIEW

File: handbook/company/remote/teammate-recommended_productivity_hacks.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ You might think of this page as a remote version of "the clever workflows and sh
 - **Automatically add links to your slack posts by copying a url, highlighting the text you want to link in the slack message, and then pasting.** This way you can use `cmd`+`v` instead of the Slack "add link" button or the slack keyboard shortcut of `cmd`+`shift`+`U`.
 - **Disable their WYSIWYG editor if you prefer markdown.** Go to Preferences > Advanced > Check "Format messages with markup."
 - **Show only unread channels in your channel sections.** Right-click on a channel section or the "Channels" header, then select "Show" > "Unread channels only."
+- **Set up keywords to get notified about relevant threads you aren't tagged in.** Go to Preferences > Notifications > My keywords and add your list of keywords separated by commas. You'll get notified whenever someone uses them.
+- **Use Slackbot's reminder feature to get reminded of threads later on without leaving them as unread** Hover on any Slack message > Select "More actions" in the tool bar > Select "Remind me about this" and choose when you'd like to be reminded. Slackbot will keep your pending messages backlog, you can mark items as Complete when you've gone through them.
```
Comment (Joelkw) [inline handbook/company/remote/teammate-recommended_productivity_hacks.md:19]:
```suggestion
- **Use [Slackbot's reminder feature](https://slack.com/help/articles/208423427-Set-a-reminder#set-a-reminder-for-a-message) to get reminded of threads later on without leaving them as unread** Hover on any Slack message > Select "More actions" in the tool bar > Select "Remind me about this" and choose when you'd like to be reminded. Slackbot will keep your pending messages backlog, you can mark items as Complete when you've gone through them. 
```

File: handbook/company/remote/teammate-recommended_productivity_hacks.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ You might think of this page as a remote version of "the clever workflows and sh
 - **Automatically add links to your slack posts by copying a url, highlighting the text you want to link in the slack message, and then pasting.** This way you can use `cmd`+`v` instead of the Slack "add link" button or the slack keyboard shortcut of `cmd`+`shift`+`U`.
 - **Disable their WYSIWYG editor if you prefer markdown.** Go to Preferences > Advanced > Check "Format messages with markup."
 - **Show only unread channels in your channel sections.** Right-click on a channel section or the "Channels" header, then select "Show" > "Unread channels only."
+- **Set up keywords to get notified about relevant threads you aren't tagged in.** Go to Preferences > Notifications > My keywords and add your list of keywords separated by commas. You'll get notified whenever someone uses them.
```
Comment (Joelkw) [inline handbook/company/remote/teammate-recommended_productivity_hacks.md:18]:
```suggestion
- **Set up [Slack keywords](https://slack.com/slack-tips/get-notified-when-someone-mentions-a-topic-you-care-about) to get notified about relevant threads you aren't tagged in.** Go to Preferences > Notifications > My keywords and add your list of keywords separated by commas. You'll get notified whenever someone uses them.
```

**Review by Joelkw — APPROVED** [review]
LGTM! Just suggested adding links to the feature docs for folks who might want to translate the language of docs or want more detail.

**Comment by macraig** [comment]
Awesome addition, thanks @Joelkw !

**Pull request merged**